code/procs/mobprocs/chatprocs.dm

```diff
@@ -544,6 +551,42 @@
 	else
 		return "[speechverb],[first_quote][font_accent ? "<font face='[font_accent]'>" : null]<span [class? class : ""]>[text]</span>[font_accent ? "</font>" : null][second_quote]"
 
+// Say emphasis stuff
+/// Transforms the speech emphasis mods from [/atom/proc/say_emphasis] into the appropriate HTML tags. Includes escaping.
+#define ENCODE_HTML_EMPHASIS(input, char, html, varname) \
+	var/static/regex/##varname = regex("(?<!\\\\)[char](.+?)(?<!\\\\)[char]", "g");\
+	input = varname.Replace_char(input, "<[html]>$1</[html]>")
+
+// Italicised text looks bad in maptext without adding phantom trailing spaces. I'm sorry. - DisturbHerb
+var/static/min_spaces_after_italics = 3 // Minimum number of trailing spaces after a portion of italicised html.
```

Comment:
Would this ever change? if there's no reasonable scenario where we want to modify this at runtime (I can't think of one, "make everyone's speech slightly uglier" isn't exactly a fun admin button), make it a define instead.
If it really has to be a var for some reason, use `global` instead of `static` for global-scoped vars.

Reply:
When goon moves off SmallFonts, i guess. Windows stopped shipping the SmallFonts font with modern distros, and its not a publically available font.

Reply:
Meant 'ever change at runtime', should have specified.

Reply:
also why static instead of const lol